addon/content/reducer.js

```diff
@@ -205,69 +190,59 @@ function messages(state = initialMessages, action) {
             tags.push(toggledTag);
           }
 
-          return browser.messages.update(action.id, {
+          return browser.messages.update(id, {
             tags,
           });
         })
         .catch(console.error);
-      return state;
-    }
-    case "MSG_STAR": {
+    };
+  },
+  msgStar({ id, star }) {
```

Comment:
This should probably be `changeStar` or `setStar` or something like that if you're happy to change it (I should have named it better previously).

Reply:
what about `starred` or `setStarred`?

Reply:
Let's go `setStarred`